picard/ui/options/dialog.py

```diff
@@ -170,6 +170,12 @@ def _restore_default(self, page):
             if option.section == 'setting':
                 old_options[option.name] = config.setting[option.name]
                 config.setting[option.name] = option.default
+
+        # Plugin manager has to be updated
```

Comment:
This change (and #551) both implement an operation to perform changes for properly resetting a page in addition to resetting the options. Instead of cluttering up this class with `if isinstance` or `if page.NAME` checks, there should be a method that the various `OptionsPage`s can implement that does this (similar to load, check, save, ...).